src/runtime.md

```diff
@@ -96,10 +38,20 @@ display a console window on startup. It will run detached from any existing cons
 
 [_MetaNameValueStr_]: attributes.md#meta-item-attribute-syntax
 [`GlobalAlloc`]: alloc::alloc::GlobalAlloc
-[`PanicInfo`]: core::panic::PanicInfo
-[abort]: ../book/ch09-01-unrecoverable-errors-with-panic.html
-[attribute]: attributes.md
 [crate types]: linkage.md
-[set_hook]: std::panic::set_hook
 [static item]: items/static-items.md
 [subsystem]: https://msdn.microsoft.com/en-us/library/fcc1zstk.aspx
+
+<script>
+(function() {
+    var fragments = {
+        "#the-panic_handler-attribute": "panic.html#the-panic_handler-attribute",
+    };
+    var target = fragments[window.location.hash];
+    if (target) {
+        var url = window.location.toString();
+        var base = url.substring(0, url.lastIndexOf('/'));
+        window.location.replace(base + "/" + target);
+    }
+})();
+</script>
```

Comment:
For my own edification: what does this do?

Reply:
It's a redirect from `runtime.httml#the-panic_handler-attribute` to `panic.html#the-panic_handler-attribute`.